lib/utils.ts

```diff
@@ -24,6 +24,7 @@ const LIBRARY_MODULES = [
   '@testing-library/svelte',
 ];
 
+// TODO: should be deleted after all rules are migrated to v4
 const hasTestingLibraryImportModule = (
```

Comment:
This method **must** disappear after refactoring all rules since this check it's part of detection module now.

Reply:
Should we mark it as `@deprecated`, because you'll get a warning if you want to use it?
Could be helpful for contributors.

Reply:
It will disappear when v4 is finished so I don't think we even need to deprecate it